inst/scripts/importPDB.R

```diff
@@ -0,0 +1,158 @@
+# importPDB.R
+# Non-exported function
+

```

Comment:
Comments look good.
You might want to keep pseudo-code in comments so file is runnable though.